foundry.toml

```diff
@@ -9,7 +9,7 @@ multiline_func_header = 'params_first_multi'
 sort_imports = true
 
 [profile.default]
-solc_version = '0.8.23'
+solc_version = '0.8.28'
```

Comment:
For my knowledge , what is the benefit of going 28 instead of 23 here? 

And should we change the upstream foundry template to reflect this?

Reply:
well no reason in particular. it's just the latest version that has bug fixes, improved gas efficiency, some new features. it's been out for like 3 months at this point. im not married to it, just thought it made sense to do